core/src/main/java/bisq/core/arbitration/DisputeManager.java

```diff
@@ -137,7 +138,7 @@ public DisputeManager(P2PService p2PService,
                           OpenOfferManager openOfferManager,
                           KeyRing keyRing,
                           PersistenceProtoResolver persistenceProtoResolver,
-                          @Named(Storage.STORAGE_DIR) File storageDir) {
+                          @Named(Storage.STORAGE_DIR) File storageDir, CorruptedDatabaseFilesHandler corruptedDatabaseFilesHandler) {
```

Comment:
Can you add a line break. So params are better separated/readable?

Reply:
sure, i wonder if there is a way to make idea do it automatically.